ex02.ex

```diff
@@ -53,7 +53,7 @@ defmodule Ex02 do
   # if the first two elements of a list are equal                #
   ################################################################
 
-  first2equal = your_anonymous_function([])
+  first2equal = fn [a,b,_c,_d,_e] -> a == b end
```

Comment:
## -4

This code only works for list containing five elements. Although this passes the test, the requirement is clearly more general.